tests/frontend/test_kibana_app/cypress/integration/step-definitions/filters/nav-to-agent-page.and.js

```diff
@@ -0,0 +1,14 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+import { clickElement, elementIsVisible} from '../../utils/driver';
+import {
```

Comment:
for sake of consistency either expand the previous imports:
import { 
    clickElement, 
    elementIsVisible
} from '../../utils/driver';

or collapse these ones
import {wazuhMenuButton, wazuhMenuButton} from '../../pageobjects/wazuh-menu/wazuh-menu.page';